src/static/styles/partials/_preview.css

```
@@ -0,0 +1,430 @@
+    object-fit: contain;
+    cursor: zoom-in;
+}
+
+.status-card.compact .image-container.native {
+    width: 100%;
+    max-width: 100%;
+    overflow: hidden;
+}
+
+/* Status card image hover feedback */
+.status-card.instance .status-body {
```

Stylelint (17.7.0)

[error] Duplicate selector ".status-card.instance .status-body", first used at line 56 (no-duplicate-selectors)

(no-duplicate-selectors)